test/test/test_execution_expectations.rb

```diff
@@ -477,6 +478,81 @@ def return_object
                     end
                 end
 
+                describe "#not_emit a generator instance" do
+                    it "validates when the event is not emitted" do
+                        plan.add(generator = EventGenerator.new)
+                        expect_execution.to { not_emit generator }
+                    end
+                    it "fails if the event is emitted within the first cycle" do
+                        plan.add(generator = EventGenerator.new)
+                        e = assert_raises(ExecutionExpectations::Unmet) do
+                            expect_execution { generator.emit }
+                                .to { not_emit generator }
+                        end
+                        assert e.message.start_with?(
+                            "1 unmet expectations\n#{generator} "\
+                            "should not be emitted, but it was:"
+                        )
+                    end
+                    it "passes by default after the first cycle" do
+                        plan.add(generator = EventGenerator.new)
+                        count = 0
+                        expect_execution
+                            .poll { count += 1; generator.emit if count > 2 }
```

Comment:
shouldn't it be `>= 2?`

Reply:
It certainly should be. Good catch.

Fixed, and changed the checks into equalities for clarity